types/echarts/echarts-tests.ts

```diff
@@ -30,6 +30,13 @@ const testAxisLabelColor = (options: echarts.EChartOption.XAxis) => {
     };
 };
 
+const testAxisLabelShowLable = (options: echarts.EChartOption.XAxis) => {
```

Comment:
Please explain why this function is required and `Lable` is probably misspelled.